scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -764,11 +764,46 @@
                                        {:job-uuid->reserved-host (apply dissoc job-uuid->reserved-host matched-job-uuids)
                                         :launched-job-uuids (into matched-job-uuids launched-job-uuids)})))
 
+(defn distribute-task-requests-to-compute-clusters
+  "Given a collection of un-matched task requests and a collection of
+  compute clusters, distributes the task requests amongst the compute
+  clusters, using a hash of the pending job's uuid."
```

Comment:
Docstring should mention it is returning a `compute-cluster->jobs` map. That is the api any future improvements need to stick to.

Reply:
Good call, thanks

Reply:
Done